source/core/CEntryPoints.cpp

```diff
@@ -118,67 +118,92 @@ VIREO_EXPORT Int32 EggShell_PokeMemory(TypeManagerRef tm,
     }
 }
 //------------------------------------------------------------
-//! Write a numeric value to a symbol. Value will be coerced as needed.
-VIREO_EXPORT void EggShell_WriteDouble(TypeManagerRef tm, const char* viName, const char* eltName, Double d)
+//! Get a reference to the type pointer and data for a symbol.
+VIREO_EXPORT EggShellResult EggShell_FindValue(TypeManagerRef tm, const char* viName, const char* eltName, TypeRef* typeRefLocation, void** dataRefLocation)
 {
-    void *pData = nullptr;
-
     SubString objectName(viName);
```

Comment:
Add TypeManagerScope (tm), before this line.

Reply:
You are right, I'll change it.